deps/thpool/thpool.c

```diff
@@ -91,6 +92,8 @@ typedef struct priority_queue {
                                        from queue */
   unsigned char n_privileged_threads; /* number of threads that always run high
                                          priority tasks */
+  atomic_uchar privilege_tickets; /* number of currently available privilege
```

Comment:
I suggest changing the terminology here, as these are no longer privileged threads, but rather threads reserved for high-priority tasks. Maybe `num_resreved_threads`

Reply:
`num_high_priority_threads` ?